etc/tempesta_fw.conf

```diff
@@ -85,6 +85,18 @@
 #   There is a special group called "default". All "server" entries defined
 #   outside an "srv_group" added to the default group implicitly.
 
+# TAG: keepalive_timeout
+#
+# Syntax:
+#   keepalive_timeout TIMEOUT
+#
+# TIMEOUT is a timeout in secounds during which a keep-alive client connection
+# will stay open on the server side for each 'listen' entry. The zero value
+# disables keep-alive client connections.
+#
```

Comment:
Here and above:

It's unclear what is `the server side`. Perhaps, this should be changed to `Tempesta`. Also, my understanding is that the option is not for `listen` entries, but rather for client connections accepted on listening ports.